tests/suite/snippets_filter_test.go

```diff
@@ -219,19 +225,15 @@ var _ = Describe("SnippetsFilter", Ordered, Label("functional", "snippets-filter
 	})
 })
 
-func waitForHTTPRouteToHaveGatewayNotProgrammedCond(httpRouteNsNames types.NamespacedName) error {
-	ctx, cancel := context.WithTimeout(context.Background(), timeoutConfig.CreateTimeout*2)
+func waitForHTTPRouteToHaveGatewayNotProgrammedCond(httpRouteNsName types.NamespacedName) error {
+	ctx, cancel := context.WithTimeout(context.Background(), timeoutConfig.RequestTimeout*2)
```

Comment:
I'd just use GetStatusTimeout instead of RequestTimeout*2, following a similar pattern in the client settings test when it gets the status.